src/app/features/pro-registration/components/service-preference/service-preference.component.ts

```diff
@@ -16,11 +19,39 @@ import { InputTextareaModule } from 'primeng/inputtextarea';
     FileUploaderComponent,
     InputTextModule,
     InputTextareaModule,
+    MultiSelectModule,
   ],
   templateUrl: './service-preference.component.html',
   styleUrl: './service-preference.component.sass',
 })
 export class ServicePreferenceComponent {
   serviceCategories = serviceCategories;
   paymentPreferences = accountPreferences;
+  bookingDays = bookingDays;
+  timeOptions: { name: string; value: string }[] = [];
```

Comment:
create reusable interface for this

```suggestion
export interface timeOptionsType {
name:string
value:string
}
  timeOptions:timeOptionsType[] = [];
```